extensions/amp-script/amp-script.md

```diff
@@ -23,37 +23,31 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# amp-script
+The `amp-script` component allows you to run custom JavaScript. To maintain AMP's performance guarantees, your code runs in a Web Worker, and certain restrictions apply.
 
-## Overview
-
-The `amp-script` component allows you run custom JavaScript to render UI elements, such as a React component.
-
-### A simple example
+## Usage
 
 An `amp-script` element can load JavaScript in two ways:
 
-- Remotely, from a URL to a JavaScript file.
-- Locally, from a `script[type=text/plain][target=amp-script]` element on the page.
+- remotely, from a URL
+- locally, from a `<script>` element on the page
 
-#### Load JavaScript from a remote URL
+#### From a remote URL
 
-Use the `src` attribute to load remote JavaScript.
+Use the `src` attribute to load JavaScript from a URL:
 
 ```html
 <amp-script layout="container" src="https://example.com/hello-world.js">
   <button>Hello amp-script!</button>
 </amp-script>
 ```
 
-If `src` points to a cross-origin URL, then a ["script hash"](#security-features) must also be added to the document head.
+#### From a local element
 
-#### Load JavaScript from a local element
-
-Use the `script` attribute to reference a local `script` element by `id`.
+Use the `script` attribute to reference a local `script` element by `id`. You must also specify `type="text/plain"` so that the browser doesn't execute your script immediately. Instead, `<amp-script>` finds it. Also include `target="amp-script"`, like this:
```

Comment:
Suggestion to update to:
```
The amp-script component can reference a local `script` element's JavaScript. You must:
- Reference the local `script` element's `id` using the `script` attribute.
- Specify the `<script>` tag as `type="text/plain"`. This stops the browser from executing your script and allows amp-script to control it.
- Include `target="amp-script"` to the `<script>` tag. 
```

Reply:
I incorporated this into another draft. WDYT?

Also, how do you feel about surrounding HTML element names in angle brackets? I usually do this. I think MDN does. But this document didn't, so I've been omitting the angle brackets for consistency's sake.

I could also just insert them whenever appropriate. As in, a `<script>` tag vs the script that `<amp-script>` runs in a Worker.